apps/site/components/Common/Skeleton/index.tsx

```diff
@@ -26,7 +27,7 @@ const Skeleton: FC<PropsWithChildren<SkeletonProps>> = ({
     <span
       tabIndex={-1}
       aria-hidden="true"
-      className={styles.skeleton}
+      className={`${styles.skeleton} ${className}`}
```

Comment:
For cases where it may be `undefined`, it may be better to use the `classNames` utility function

Reply:
Yup, agreed, use classNames